python/plugins/processing/modeler/ModelerDialog.py

```diff
@@ -646,9 +646,7 @@ def _addAlgorithm(self, alg, pos=None):
         dlg = None
         try:
             dlg = alg.getCustomModelerParametersDialog(self.model)
```

Comment:
I wonder if we should kill this all together.
No algorithm implements this and I wonder if it's even feasible to implement a dialog that mimics the behavior of `ModelerParametersDialog`.

Reply:
I'm not sure either - but it's fixed API now

Reply:
Ah you're right - I was getting confused with another method. I think I did this try/catch originally because I didn't want to add getCustomModelerParametersDialog to the core QgsProcessingAlgorithm API... but I'm not sure which algorithm was originally using it and when that algorithm was changed. I think it can be safely dropped now too.